compute/rn/release-information/release-notes-22-06-update2.adoc

```diff
@@ -0,0 +1,111 @@
+:toc: macro
+== 22.06 Update 2 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.06.xxx
+
+|Codename
+|Kepler, 22.06 Update 2
+//Tentative date
+|Release date
+|Sep 15, 2022
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+toc::[]
+
+=== Enhancements
+// GithubIssue PCSUP-issue#
+
+// GH#39229
+==== HTTPS Proxy Support for Agentless Scanning
+
+* Agentless scanning now supports connections over an HTTPS proxy server.  If you use custom certificates for authentication, you can now configure custom certificates for the connection to Console when using https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/configure/configure-agentless-scanning[agentless scanning].
+
+// GH# 39754
+==== Support for Embedding a Defender in a CloudFormation Fargate Task
```

Comment:
@ssugandh I think this title is confusing, because we already support it, in Kepler Update 2 we enhanced it.